src/test/javascript/spec/component/iris/http-message.service.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { TestBed, fakeAsync, tick } from '@angular/core/testing';
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
+import { take } from 'rxjs/operators';
+import { mockClientMessage, mockConversation, mockServerMessage } from '../../helpers/sample/iris-sample-data';
+import { IrisHttpMessageService } from 'app/iris/http-message.service';
+import { IrisClientMessage } from 'app/entities/iris/iris-message.model';
+
+describe('HTTP Message Service', () => {
```

Comment:
For consistency with `Iris Http Session Service` in the session test file
```suggestion
describe('Iris Http Message Service', () => {
```